package.json

```diff
@@ -26,7 +26,7 @@
     "update-snapshots": "yarn cmsds test ./packages --updateSnapshot",
     "test-coverage": "yarn cmsds test ./packages --collectCoverage",
     "posttest": "yarn lint",
-    "lint": "yarn cmsds lint ./ --ignorePatterns '**/node_modules/**' '**/dist/**' '**/helpers/**' '**/__tests__/**' 'tmp/**' '**/types/**' 'examples/create-react-app-typescript/**' 'examples/create-react-app/**' 'storybook-static/**' 'packages/coverage/**'",
+    "lint": "yarn cmsds lint ./ --ignorePatterns '**/node_modules/**' '**/dist/**' '**/helpers/**' '**/__tests__/**' 'tmp/**' '**/types/**' 'examples/create-react-app-typescript/**' 'examples/create-react-app/**' 'storybook-static/**' 'packages/coverage/**' 'packages/design-system-tokens/src/lib/types.ts'",
```

Comment:
I've submitted a [PR](https://github.com/TheDancingCode/gulp-prettier/pull/68) to them to bump this so hopefully can remove this in the future